### DISCUSSION TITLE
Test suite/harness

### DISCUSSION THREAD

**Discussion by habere-et-dispertire** [tonsky/FiraCode#1432] [Q&A]
Is there a test suite?

I don't think I've ever seen such a thing for a typeface!

**Comment by tonsky** [comment]
Not currently. How do you see it?

**Comment by habere-et-dispertire** [comment]
I do not know. I'm looking into visual testing with https://percy.io/

It seems to use a web harness. So a set of glyphs each on its own page might be a way to achieve this.
